counsel.el

```diff
@@ -2063,6 +2063,65 @@ By default `counsel-bookmark' opens a dired buffer for directories."
    ("r" ,(counsel--apply-bookmark-fn #'counsel-find-file-as-root)
         "open as root")))
 
+;;** `counsel-bookmarked-directory'
+(defcustom counsel-bookmarked-directory-default-name "%s"
+  "Default bookmark name created by `counsel-bookmarked-directory-add."
```

Comment:
Neither the variable name nor the docstring make it clear that the value should/can be a format string. I'm not familiar with how it's used yet, but at first glance a name like `counsel-bookmarked-directory-format` or similar would seem more intuitive.

Reply:
I simply omitted this custom variable. Thanks.